src/User.js

```diff
@@ -55,16 +57,24 @@ class User {
   // }
 
   searchSaved(searchItem) {
-   return this.favRecipes.filter(recipe => {
-      return recipe.name.includes(searchItem)    
-
-      || recipe.ingredients.find(ingredient => {
-        return ingredient.name.includes(searchItem)
+  // this.favRecipes.filter(recipe => {
+  //     return recipe.name.includes(searchItem)    
```

Comment:
I think we can get this in the same function. But if not then we may have to split up up into two different methods